documentation/modules/auxiliary/scanner/http/title.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+Generates a GET request to the provided web servers and returns the server header, HTML title attribute and location header (if set). This is useful for rapidly identifying interesting web applications en mass.
+
+## Verification Steps
+
+  1. Do: `use auxiliary/scanner/http/title`
+  2. Do: `set rhosts [ips]`
+  3. Do: `run`
+
+## Options
+
+**SHOW_TITLES**
+
+If set to `false`, will not show the titles on the console as they are grabbed. Defaults to `true`.
+
+**STORE_NOTES**
+
+If set to `false`, will not store the captured information in notes. Use "notes -t http.title" to view. Defaults to `true`.
```

Comment:
Nice addition with the command to view the notes!
Wrap it in ticks instead of double quotes though.

Reply:
Done.